.github/ISSUE_TEMPLATE/deprecation_template.md.txt

```diff
@@ -0,0 +1,23 @@
+---
+name: Deprecation Notice
+about: A notice for one or more feature or API deprecations
+title: 'Deprecation Notice'
+labels: 'needs-triage,needs-sig,kind/deprecation'
+
+---
+
+## Deprecated APIs
+List the fully qualified APIs/types. For example, `AZ::Quaternion::SetFromEulerRadians(const Vector3&)`
+
+## Alternatives APIs
+List new or updated APIs to replace deprecated API. 
+For example, use 
+`Quaternion AZ::Quaternion::CreateFromEulerRadiansXYZ(const Vector3&)` 
+to repleace 
+`void AZ::Quaternion::SetFromEulerRadians(const Vector3&)` and `Quaternion AZ::Quaternion::ConvertEulerRadiansToQuaternion(const Vector3&)`
+
+## Last Release
+Describe the last release version before this deprecation. For example: `stabilization/2210`
```

Comment:
A quick question: would o3de-extras sync releases with O3DE, or extra Gems be versioned independently? If so, perhaps this should be adjusted.

Reply:
I recall having discussions about this at sig-release. @tonybalandiuk do you know the latest status update on this decision?

Reply:
I am looking forward to the decision from sig-release. Generally, if releases were not synchronized, there would have to be some kind of structured mechanism for Gems to specify version compatibility to allow (optimally automatic) selection of the "correct" (likely latest compatible) gem version in the project manager. I am not aware of any such mechanism at this time.